profile/README.md

```diff
@@ -1 +1,8 @@
-Welcome!
+## Welcome to the OpenVINO™ GitHub organization!
+[OpenVINO™](https://en.wikipedia.org/wiki/OpenVINO) is an open-source toolkit for optimizing and deploying AI inference.
```

Comment:
I think we should use either https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/overview.html or https://docs.openvino.ai/2023.0/home.html here (probably intel.com to avoid issues with doc versions)

Reply:
I fully agree. We can use "docs.openvino.ai" as a safe reference to the most recent version.